### ISSUE TITLE
Doesn't actually copy built content to cordova folder if already created

### ISSUE THREAD

**Issue by RangerMauve** [SamVerschueren/gulp-cordova-create#10]
The data being piped from `gulp.src` isn't being piped into the `www` folder if the project has already been created. Currently I have to set up a separate task to do that and have the creation just done at project creation.

**Comment by SamVerschueren** [comment]
You're right. Personally I recreate my cordova project every time. But that's probably just me. It would be a nice addition if this was supported though. Can't look into it at the moment but a PR is always welcome :).
